app-modules/task/src/Imports/TaskImporter.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Assist\Task\Imports;
+
+use App\Models\User;
+use App\Models\Import;
+use App\Imports\Importer;
+use AllowDynamicProperties;
+use Illuminate\Support\Str;
+use Assist\Task\Models\Task;
+use Assist\Task\Enums\TaskStatus;
+use Assist\Prospect\Models\Prospect;
+use Illuminate\Validation\Rules\Enum;
+use Illuminate\Database\Eloquent\Builder;
+use Assist\AssistDataModel\Models\Student;
+use App\Filament\Actions\ImportAction\ImportColumn;
+
+/**
+ * @property ?Task $record
+ */
+#[AllowDynamicProperties]
+class TaskImporter extends Importer
+{
+    /**
+     * @inheritDoc
+     */
+    public static function getColumns(): array
+    {
+        return [
+            ImportColumn::make('title')
+                ->requiredMapping()
+                ->example('The task title'),
+            ImportColumn::make('description')
+                ->requiredMapping()
+                ->example('A description of the task.'),
+            ImportColumn::make('status')
+                ->requiredMapping()
+                ->rules([new Enum(TaskStatus::class)])
+                ->example('pending'),
+            ImportColumn::make('due')
+                ->rules(['date_format:Y-m-d H:i:s'])
+                ->example(fake()->dateTimeBetween('+1 day', '+1 week')->format('Y-m-d H:i:s')),
```

Comment:
I believe this will fail in production because `composer install --no-dev` will not install faker.